.circleci/config.yml

```diff
@@ -384,32 +384,36 @@ commands:
-            # Create list of kube versions
-            CURRENT_KUBE_VERSIONS=$(curl -L https://raw.githubusercontent.com/kubernetes/website/main/data/releases/schedule.yaml \
-              | yq -o json '.' \
-              | jq --raw-output '.schedules[] | select((now | strftime("%Y-%m-%dT00:00:00Z")) as $date | .releaseDate < $date and .endOfLifeDate > $date) | .previousPatches[0].release')
-
-            # Use helm to template our chart against all kube versions
-            TEMPLATE_DIR=$(mktemp -d)
-            for kube_version in ${CURRENT_KUBE_VERSIONS}; do
-              # Use helm to template our chart against kube_version
-              helm template --kube-version "${kube_version}" router helm/chart/router --set autoscaling.enabled=true > "${TEMPLATE_DIR}/router-${kube_version}.yaml"
-
-              # Execute kubeconform on our templated charts to ensure they are good
-              "${KUBECONFORM_INSTALL}/kubeconform" \
-                --kubernetes-version "${kube_version}" \
-                --strict \
-                --schema-location default \
-                --verbose \
-                "${TEMPLATE_DIR}/router-${kube_version}.yaml"
-            done
+            # As of 2023-09-20, no current kube versions are supported by kubeconform,
+            # due to https://github.com/yannh/kubeconform/issues/233. So we skip
+            # this check for now.
+            #
+            # # Install Helm
+            # curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+            # # Install kubeconform
+            # KUBECONFORM_INSTALL=$(mktemp -d)
+            # curl -L https://github.com/yannh/kubeconform/releases/latest/download/kubeconform-linux-amd64.tar.gz | tar xz -C "${KUBECONFORM_INSTALL}"
+
+            # # Create list of kube versions
+            # CURRENT_KUBE_VERSIONS=$(curl -L https://raw.githubusercontent.com/kubernetes/website/main/data/releases/schedule.yaml \
+            #   | yq -o json '.' \
+            #   | jq --raw-output '.schedules[] | select((now | strftime("%Y-%m-%dT00:00:00Z")) as $date | .releaseDate < $date and .endOfLifeDate > $date) | .previousPatches[0].release')
+
+            # # Use helm to template our chart against all kube versions
+            # TEMPLATE_DIR=$(mktemp -d)
+            # for kube_version in ${CURRENT_KUBE_VERSIONS}; do
+            #   # Use helm to template our chart against kube_version
+            #   helm template --kube-version "${kube_version}" router helm/chart/router --set autoscaling.enabled=true > "${TEMPLATE_DIR}/router-${kube_version}.yaml"
+
+            #   # Execute kubeconform on our templated charts to ensure they are good
+            #   "${KUBECONFORM_INSTALL}/kubeconform" \
+            #     --kubernetes-version "${kube_version}" \
+            #     --strict \
+            #     --schema-location default \
+            #     --verbose \
+            #     "${TEMPLATE_DIR}/router-${kube_version}.yaml"
+            # done
```

Comment:
https://github.com/apollographql/router/pull/3867#pullrequestreview-1637280844 addresses this (workaround)